setup.cfg

```diff
@@ -12,8 +12,7 @@ packages = find:
 # Parse the MANIFEST.in file and include those files, too.
 include_package_data = True
 # Let pip install dependencies automatically.
-install_requires = beaker-client
-                   flake8
+install_requires = flake8
```

Comment:
This is a bummer. Do we have to break the pip package dependencies for Circle CI?